mathics/builtin/__init__.py

```diff
@@ -5,34 +5,25 @@
 Mathics has over a thousand Built-in Functions and variables, all of which are defined here.
 """
 
-from typing import Optional
 import glob
 import importlib
 import inspect
+import os.path as osp
 import pkgutil
 import re
-import os.path as osp
+from typing import List, Optional
 
 from mathics.builtin.base import (
     Builtin,
-    SympyObject,
     Operator,
     PatternObject,
+    SympyObject,
+    mathics_to_python,
 )
-
 from mathics.core.pattern import pattern_objects
-
 from mathics.settings import ENABLE_FILES_MODULE
 from mathics.version import __version__  # noqa used in loading to check consistency.
 
-from typing import List
```

Comment:
Although this is moved later below right now. All of this should not be in in `__init__.py` 

`__init__.py` should be small.